x-pack/plugins/rollup/server/collectors/register.ts

```diff
@@ -208,8 +53,10 @@ export function registerRollupUsageCollector(
       visualizations: {
         saved_searches: {
           total: { type: 'long' },
+          lens_total: { type: 'long' },
         },
         total: { type: 'long' },
+        lens_total: { type: 'long' },
```

Comment:
do you mind describing the fields? you can do so by using the `_meta.description` properties like in the example:

```
schema: {
  ...
  my_field: {
    type: 'long',
    _meta: { description: 'Counts all the usages of my_field in...' }
  }
}
```

Reply:
Done @afharo https://github.com/elastic/kibana/pull/98639/commits/786a23271da8b0b25c89610315fe99c05cf59c6d